soccer/gameplay/skills/pivot_kick.py

```diff
@@ -66,7 +66,7 @@ def __init__(self):
             'fumble')
 
         # default parameters
-        self.dribbler_power = constants.Robot.Dribbler.MaxPower
+        self.dribbler_power = 60#constants.Robot.Dribbler.MaxPower
```

Comment:
IMO this should be chnaged in the config xml (via dribbler multiplier), can we leave this at 100% and retune this later). As of now, this is affecting the simulator too...